cloud-service-providers/azure/azureml/cli/1_set_credentials.sh

```diff
@@ -2,18 +2,46 @@
 set -x
 source config.sh
 
+CREATE_RESOURCE_GROUP=false
+CREATE_CONTAINER_REGISTRY=false
 CREATE_WORKSPACE=false
 
 for i in "$@"; do
   case $i in
-    --create_new_workspace) CREATE_WORKSPACE=true ;;
-    -*|--*) echo "Unknown option $i"; exit 1 ;;
+    --create_new_resource_group)
```

Comment:
Can you remove the create resource group flag? I think it is safe to assume that users will already have a resource group to deploy this on.

Reply:
I actually dont think it is safe.

Its a bit redundant but it makes the script to work even when the user does not have a work group for the workspace

I for instance like to create a worksgroup for every single project/test I do as it is way easier to delete resources by workgroup than individually